src/main/java/apoc/metrics/Metrics.java

```diff
@@ -147,9 +74,35 @@ public static StorageMetric fromStoragePair(StoragePair storagePair) {
         }
     }
 
+    /**
+     * DAO for a single line in a metrics/*.csv file.
+     * Value is abstracted to double; in reality some are int, some are float, some are double.
+     */
+    public static class GenericMetric {
+        public final long timestamp;
+        public final String metric;
+        public final Map<String,Object> map;
+
+        public GenericMetric(String metric, long t, Map<String,Object> map) {
```

Comment:
This DAO is about all that is generic to all metrics.

The "value" field is not generic.  It's in some but not in others.

Keeping in mind -- timestamp is UNIX time, not necessarily in UTC, but in the timezone the database is configured to use (which is usually UTC).  I'm leaving it as its published by the database in order to not deal with issues like fetching the configured TZ prior to getting the metric, which would allow me to convert to a timestamp, but would only then create the secondary issue of what TZ to represent the output in (is database timezone or UTC better?  Hmm.....)

Reply:
probably a choice for the user on the UI?